### ISSUE TITLE
Can't Release Token When Moving Them

### ISSUE THREAD

**Issue by Justeee** [dan-cave/gaming-table#1]
When I'm trying to move a token on the map using the token select tool, I'm able to drag them just fine but when I remove my finger from the touch screen/IR frame, it doesn't read that the token has been released and to place them there. Instead, they hover there and there's a shadow/translucent version of that token in their original position. 

Everything else works GREAT and I love the tapping instead of pinching and panning but this is the one hiccup I'm hitting.

**Comment by dan-cave** [comment]
Hi @Justeee , Someone caught this bug a few days ago, so I'm actually working on a fix right now. I could use your help tracking down the cause of this bug for you, so I can see if it matches the way I'm reproducing it locally. I've only reproduced this on a laptop attached to a monitor with an IR frame (so not a native touchscreen/android/apple device).

This is how I've figured out how to reproduce this bug:

1. Open foundry on your IR Frame gaming table client
2. log in to foundry
3. Allow the scene to load
4. Open the devtools (F12).
5. If the device toolbar is not toggled, toggle it with ctrl + shift + M
6. attempt to move a token. You shouldn't be able to drop after dragging.


https://github.com/dan-cave/gaming-table/assets/20744784/8d9e19da-47d9-4d25-95b1-05c41309ad7e


TL;DR: This bug arises for me when I load the canvas without touch emulation enabled in chrome.

**The current workaround I've found is to refresh the browser WITH touch emulation enabled. I haven't been able to recreate this bug when the canvas loads with touch emulation enabled.**

If you've seen this bug on a touch enabled device (or an android/apple device), or when the canvas loads with touch emulation enabled please let me know. It'll help me track down the real source of this bug.


I'm currently working on a fix for this bug. In the mean time give the workaround I've bolded above a try. If that doesn't work let me know so we can try to find another one.

Edit: Also, thanks for using my module! I appreciate the feedback.

**Comment by OregonPinkRose** [comment]
Using a touchscreen laptop, I'm also having the "can't let go" bug.

**Comment by dan-cave** [comment]
> Using a touchscreen laptop, I'm also having the "can't let go" bug.

Hi @OregonPinkRose , thanks for reaching out. I don't have access to a desktop/laptop with a native touchscreen input, so if you're willing to provide some assistance and answer the following questions I'd really appreciate it.

- What browser are you currently using?
- Do you have touch emulation enabled in the browser you're using? 
  - I don't think the foundry desktop/electron client supports touch emulation, so if you're using that, then the answer would be no.
- If not, do you know if your touches are actually firing touch events in the browser you're using?
  - I haven't found an answer to this question yet, but I'm not sure if the Desktop chrome browser actually triggers touch events from native touch screen devices. IR Frames seem to trigger mouse events when tapped.
  -  If you visit this webpage, https://dan-cave.github.io/touchTest/, tap anywhere on the page, and see a popup box saying you triggered a "touchstart" event, then you are tiggering touch events. If not then you're probably triggering mouse events.

Any help would be greatly appreciated.

**Comment by OregonPinkRose** [comment]
Chrome 119.0.6045.106
I could not find any setting for touch emulation enabling. I found a few how-to websites, but they were older.
On my touchscreen laptop, I do get the pop-up that says I triggered a touch event.
As expected, on my desktop, I do not get any po-ups with mouse clicks.
Happy to help with anything else.

**Comment by dan-cave** [comment]
Hi everyone.

I just released a new version of this module, v1.1.0, that has fixed the issues _I've seen_ with failures to finish a drag event on desktop browsers using touch emulation and on native touchscreen android browsers using.

If you get a chance to upgrade and see any improvements (and especially if not), I'd love to here back.

Thanks
